Tests/UnitTests/Application/MainCoordinatorTests.swift

```diff
@@ -249,13 +273,18 @@ extension MainCoordinatorTests {
         XCTAssertEqual(mockAnalyticsService.eventsParamsLogged["text"], iconEvent.text)
         XCTAssertEqual(mockAnalyticsService.additionalParameters["taxonomy_level2"] as? String, "login")
         XCTAssertEqual(mockAnalyticsService.additionalParameters["taxonomy_level3"] as? String, "undefined")
+        
+        UserDefaults.standard.removeObject(forKey: FeatureFlags.enableWalletVisibleToAll.rawValue)
     }
     
     @MainActor
     func test_didSelect_tabBarItem_profile() {
+        UserDefaults.standard.set(false, forKey: FeatureFlags.enableWalletVisibleToAll.rawValue)
+        UserDefaults.standard.set(false, forKey: "hasAccessedWalletBefore")
+        
```

Comment:
Can we use `AppEnvironment.updateReleaseFlags` here as we have in other places

Reply:
So for some reason, when using `AppEnvironment.updateReleaseFlags` locally, the tests do not pass consistently..? Happy to have a proper discussion about it, I could be doing something incorrectly elsewhere.